src/main/cc/wfa/measurement/internal/duchy/protocol/liquid_legions_v2/liquid_legions_v2_encryption_utility.cc

```diff
@@ -289,6 +295,8 @@ absl::StatusOr<int64_t> AddBlindedHistogramNoise(
       }
     }
   }
+
+  delete pDistributedRandomNoise;
```

Comment:
avoid using raw pointer, i.e. no `new/delete` in the code.

either just use an instance, or use smart pointers